src/pages/publication/contributors_tab/components/AffiliationsCell.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FC, useState } from 'react';
+import { Institution } from '../../../../types/contributor.types';
+import { Button } from '@material-ui/core';
+import SelectInstitution from '../../../../components/SelectInstitution';
+import Modal from '../../../../components/Modal';
+import { useFormikContext, FormikProps } from 'formik';
+import { FormikPublication, BackendTypeNames } from '../../../../types/publication.types';
+import { SpecificContributorFieldNames } from '../../../../types/publicationFieldNames';
+import { FormikInstitutionUnit } from '../../../../types/institution.types';
+import { useTranslation } from 'react-i18next';
+import NormalText from '../../../../components/NormalText';
+import styled from 'styled-components';
+import ConfirmDialog from '../../../../components/ConfirmDialog';
+import AddIcon from '@material-ui/icons/Add';
+import DeleteIcon from '@material-ui/icons/Delete';
+import { publicationLanguages, LanguageCodes } from '../../../../types/language.types';
+
+const StyledAffiliationsCell = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const StyledRemoveAffiliationButton = styled(Button)`
+  margin-left: 1rem;
+`;
```

Comment:
<img width="455" alt="Skjermbilde 2020-04-20 kl  14 14 56" src="https://user-images.githubusercontent.com/6595982/79750548-70e59c00-8311-11ea-9f6a-e90edb245be2.png">

Syns ikke dette ser helt bra ut. Navn på institutt og fjern-knappen bør være sentrert vertikalt, og synes det bør være større forskjell på "Legg til tilknytning" og teksten med navnet på instituttet. Burde vært mer spacing mellom tekst og knapp også synes jeg

Reply:
Sentrering er ikke all verdens nei :P 
Vil egentlig ikke at de skal være for "knappete" heller, for det blir fort veldig mye støy med alskens knapper/actions når antall contributors øker.
Skal se litt mer på det!

Reply:
![Skjermbilde 2020-04-22 kl  10 33 47](https://user-images.githubusercontent.com/6595982/79959772-d017ed80-8484-11ea-9a55-ee1c3702a38c.png)

Synes fortsatt det ser litt rart ut, men tenker at det heller får være designeren sin jobb å fikse dette 😅

Reply:
nå viser vi bare løvnoden også da. Når oppslag for subunit er klart må presentasjonen av hver affiliation endres til å vise hierarki uansett, så dette er veldig tentativt